lib/routes.js

```diff
@@ -48,7 +48,7 @@ module.exports = ({ cache, config }) => {
     } else if (userAgent.isMac && !isUpdate) {
       platform = 'dmg'
     } else if (userAgent.isWindows) {
-      platform = 'exe'
+      platform = 'msi'
```

Comment:
This is the only actual change in here - everything else is auto-formatting changes.